src/less/chat.less

```diff
@@ -10,7 +10,14 @@
   font-size: 13px;
   position: relative;
 }
-
+#chat-log .message.whisper {
+  background: #e8e8ef;
+  border: 5px solid #545469;
+}
+#chat-log .message.blind {
+  background: #ffebff;
```

Comment:
Same as above, use `background-color` instead of just `background`. 

The color here is better, since it stands out and the chat card is obviously different from the norm. We might play with it though, and see if we can find a better color. Overall, this one is fine, though.

Reply:
https://colour-palette-generator.azurewebsites.net/?r=255&g=202&b=253 is more complementary to the header color.